Submitted By: Jim Gifford <jim at linuxfromscratch dot org>
Date: 2006-07-04
Initial Package Version: 0.3.18
Origin: Various
Description: Make libol-config Multilib Friendly

diff -Naur libol-0.3.18.orig/libol-config.in libol-0.3.18/libol-config.in
--- libol-0.3.18.orig/libol-config.in   1999-07-10 06:23:07.000000000 -0700
+++ libol-0.3.18/libol-config.in        2006-08-17 22:03:38.523363309 -0700
@@ -47,8 +47,7 @@
       echo $includes
       ;;
     --libs)
-      libdirs=-L@libdir@
-      echo $libdirs -lol @LIBS@
+      echo -lol @LIBS@
       ;;
     --static-libs)
       echo @libdir@/libol.a @LIBS@

